web/src/utilities/utiltyFunctions.ts

```diff
@@ -49,3 +51,56 @@ export function opacityForZoomLevel(currentZoom: number, maxZoom: number): numbe
   }
   return maxOpacity - (0.4 * diff);
 }
+
+export function centroidStringToCoords(input: string): Array<number> {
+  const centroidString = input.match(/\(([0-9|.|\s])+\)/);
+  if (!centroidString) return [-1, -1];
```

Comment:
Could you put this line in brackets? It would make it more readable.
```suggestion
  if (!centroidString) { 
    return [-1, -1];
  }
```

Reply:
Changed in [0f3f8b2](https://github.com/atlascope/atlascope/pull/193/commits/0f3f8b2f095924f12167cdf74d4ee27a9ffc0ab4)